### DISCUSSION TITLE
Brightness control are not working on my Samsung LF32TU87 (Hardware dimming)

### DISCUSSION THREAD

**Discussion by gagarine** [MonitorControl/MonitorControl#1314] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Brightness control are not working on my Samsung LF32TU87.

Software dimming does works but not hardware dimming.

### Expected behavior

Brightness control change the brightness hardware value of my Samsung LF32TU87.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.1 (22C65)
- Mac model: M1 Macbook Air
- MonitorControl version:4.1.0
- Monitor(s): Samsung LF32TU87
- Monitor Cable(s)/Connection(s): USB-C DisplayPort
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by gagarine** [comment]
Their is another issue for this monitor https://github.com/MonitorControl/MonitorControl/issues/1042 but no one mentionnes a problem with hardware dimming. Actually it seams to works for peoples (but perhaps it was only the software dimming without realizing it didn't works)

**Comment by gagarine** [comment]
The problem is not with monitorcontrol. It's because brightness control is disable with "eco mode" is on. Disabling the eco mode solve the problem.

**Comment by willxie** [comment]
I have the same monitor with M1 MBP16" and the same issue. Disabling "Eco Saving Plus" mode unlocks OSD brightness settings. However, monitorcontrol cannot adjust the brightness for this monitor still. The brightness bar would show up but noop to the screen and real settings. @gagarine Could you share more on how you fixed this?

**Comment by gagarine** [comment]
It works for a moment... and then it failed again. I need to investigate to understand what's the problem.

**Comment by waydabber** [comment]
I'll move this to discussions as it is a valid, but display specific issue which cannot be solved by changing how MonitorControl works.
